scripts/ci/jobs/gke_ui_e2e_tests.py

```diff
@@ -17,6 +17,7 @@
 # Override test env defaults here:
 # (for defaults see: tests/e2e/lib.sh export_test_environment())
 os.environ["OUTPUT_FORMAT"] = "helm"
+os.environ["SENSOR_SCANNER_SUPPORT"] = "true"
```

Comment:
is this new? I never heard of it before

Reply:
@dcaravel turned me on to it. It triggers the install of a Secured Cluster scanner, to be used for delegating scanning duties to the cluster where the image is used, or for scans invoked manually through roxctl, when the customer has locked down their private registry/registries to only be accessible from certain clusters.